src/schema/posts.ts

```diff
@@ -1749,6 +1769,57 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
         true,
       );
     },
+    fetchSmartTitle: async (
```

Comment:
Isn't it actually enough if this field exist on post graphql, you can just query POST buy only request set field?

Reply:
It could be but it would be harder to do this logic inside the field.

Reply:
don't have preference on one or the other

Reply:
That's fine with me, if it's a speed/DX issue let's keep as is.

My initial concern was how to do the opposite side (already translated -> original)
but think we should include that either way for SEO, so let's keep this query.